### PR TITLE
ISPN-12130 RollingUpgradeIT ignoring system properties

### PR DESCRIPTION
**Pull request by ryanemerson**
https://issues.redhat.com/browse/ISPN-12130

### CODE REVIEW

File: server/tests/src/test/java/org/infinispan/server/functional/RollingUpgradeIT.java [hunk 1]
```diff
@@ -171,7 +170,7 @@ private void createSourceClusterCache(RestClient client) {
 
    private static class ClusterConfiguration extends InfinispanServerTestConfiguration {
       public ClusterConfiguration(String configurationFile, int numServers, int portOffset) {
-         super(configurationFile, numServers, ServerRunMode.EMBEDDED, new Properties(), null, null,
```
Comment (danberindei) [inline server/tests/src/test/java/org/infinispan/server/functional/RollingUpgradeIT.java:174]: Isn't this kind of excessive?

Same with https://github.com/infinispan/infinispan/blob/b23ab3509ab93be12d820fdadc5be350fb772cca/server/testdriver/core/src/main/java/org/infinispan/server/test/core/AbstractServerConfigBuilder.java#L33

Comment (ryanemerson) [reply server/tests/src/test/java/org/infinispan/server/functional/RollingUpgradeIT.java:174]: What do you propose?

**Comment by danberindei** [comment]
The new version looks better IMO.

Would you mind also changing `AbstractServerConfigBuilder`?

**Comment by ryanemerson** [comment]
> Would you mind also changing `AbstractServerConfigBuilder`?

Done. I have left this as importing all of "org.infinispan" for now.

**Comment by danberindei** [comment]
Merged, sorry for the delay Ryan!

**Pull request merged**